client/src/interface.rs

```diff
@@ -33,7 +33,7 @@ use p2p::{identity::Keypair, DialErr, InitKeypair, ListenErr, ListenRelayErr, Ou
 
 use actix::prelude::*;
 use serde::{Deserialize, Serialize};
-use std::{path::PathBuf, time::Duration};
+use std::{convert::TryInto, path::PathBuf, time::Duration};
```

Comment:
```suggestion
use std::{path::PathBuf, time::Duration};
```
`TryInto` is already imported by default

Reply:
`TryInto` is only in the prelude starting in Edition 2021. We are currently still using edition 2018: https://github.com/iotaledger/stronghold.rs/blob/312742c212c140adb0700e40d9af63af86da185e/client/Cargo.toml#L8

I think we should upgrade to edition 2021, but imo that should be done in a separate issue/ PR.